roles/openshift_manage_node/tasks/main.yml

```diff
@@ -1,3 +1,13 @@
+- name: Wait for Node Registration
+  delegate_to: "{{ openshift_first_master }}"
```

Comment:
delegate_to is not needed here, the the playbook containing the role is set for oo_first_master already.